addons/account/static/src/components/many2x_tax_tags/many2x_tax_tags.js

```diff
@@ -47,6 +47,17 @@ export class Many2XTaxTagsAutocomplete extends Many2XAutocomplete {
         });
     }
 
+    async search(name) {
+        const records = await this.orm.call(this.props.resModel, "search_read", [], {
+            domain: [...this.props.getDomain(), ["name", "ilike", name]],
+            fields: ["id", "name"],
+        });
```

Comment:
We could call `searchRead` instead, like it's done elsewhere in the codebase :)

```suggestion
        const records = await this.orm.searchRead(
            this.props.resModel,
            [...this.props.getDomain(), ["name", "ilike", name]],
            ["id", "name"],
        );
```